addons/pinniped/config-controller/README.md

```diff
@@ -15,4 +15,18 @@ $ ./hack/test.sh
 To run static checks (e.g., `go fmt`, `go vet`, `golangci-lint`) and the tests:
 ```sh
 $ ./hack/check.sh
-```
\ No newline at end of file
+```
+
+To generate the default pinniped addon secret:
+
+```sh 
+# manually via ytt
+ytt -f ytt/default-package-secret.yaml -f ytt/values.yaml -v tkr=v1.23.3---vmware.1-tkg.1 -v iaas=vsphere
+
+# using the generate script
+./generate-package-secret.sh -v tkr=v1.23.3---vmware.1-tkg.1 -v iaas=vsphere 
```

Comment:
use path from root of `config-controller` directory like other examples in README? i.e., `./hack/generate-package-secret.sh`?

Reply:
will do.